sdk/iot/provisioning/inc/az_iot_provisioning_client.h

```diff
@@ -251,18 +259,63 @@ typedef struct az_iot_provisioning_client_register_response
  * @param[out] out_response If the message is register-operation related, this will contain the
  *                          #az_iot_provisioning_client_register_response.
  * @return #az_result
- *         - #AZ_ERROR_IOT_TOPIC_NO_MATCH if the topic is not matching the expected format.
+ *         - `AZ_ERROR_IOT_TOPIC_NO_MATCH` if the topic is not matching the expected format.
  */
 AZ_NODISCARD az_result az_iot_provisioning_client_parse_received_topic_and_payload(
     az_iot_provisioning_client const* client,
     az_span received_topic,
     az_span received_payload,
     az_iot_provisioning_client_register_response* out_response);
 
+/**
+ * @brief Azure IoT Provisioning Service operation status.
+ * 
+ */
+typedef enum
+{
+  // Device assignment in progress.
+  AZ_IOT_PROVISIONING_STATUS_UNASSIGNED,
+  AZ_IOT_PROVISIONING_STATUS_ASSIGNING,
+  
+  // Device assignment operation complete.
+  AZ_IOT_PROVISIONING_STATUS_ASSIGNED,
+  AZ_IOT_PROVISIONING_STATUS_FAILED,
+  AZ_IOT_PROVISIONING_STATUS_DISABLED,
+} az_iot_provisioning_client_operation_status;
+
+/**
+ * @brief Returns the #az_iot_provisioning_client_operation_status of a
+ * #az_iot_provisioning_client_register_response object.
+ *
+ * @param[in] response The #az_iot_provisioning_client_register_response obtained after a successful
+ *                     call to #az_iot_provisioning_client_parse_received_topic_and_payload.
+ * @param[out] out_operation_status The registration operation status.
+ * @return #az_result
+ *         - #AZ_ERROR_PARSER_UNEXPECTED_CHAR if the string contains an unexpected value.
+ *         @remark See #az_iot_provisioning_client_register_response for acceptable values.

```

Comment:
I don't believe this will render as intended. Move the `@remark` after the @brief in its own standalone line, if it isn't important that it be connected to and display next to the `AZ_ERROR_PARSER_UNEXPECTED_CHAR` result.

Reply:
You're right: it renders at top-level, not within the return paragraph.

Reply:
On second thought, I'm removing this completely since the value comes from a struct that we've parsed and it's unclear.